homeassistant/components/google_translate/tts.py

```diff
@@ -129,7 +128,7 @@ async def async_get_tts_audio(self, message, language, options=None):
             url_param = {
                 "ie": "UTF-8",
                 "tl": language,
-                "q": yarl.URL(part).raw_path,
+                "q": part,
```

Comment:
I'm surprised this was ever necessary, given that `aiohttp` should do this for all data passed to `params`